app/views/provider/admin/backend_apis/mapping_rules/new.html.slim

```diff
@@ -0,0 +1,5 @@
+= content_for :sublayout_title, 'New Mapping Rule'
```

Comment:
This is not showing up:

![Screen Shot 2019-08-13 at 2 39 04 PM](https://user-images.githubusercontent.com/1842261/62942242-2d2cb100-bdd8-11e9-9841-06fc976abd7c.png)

Reply:
@guicassolato what do you think to add an `h2`?

Reply:
🤷‍♂️